picard/script.py

```diff
@@ -1195,10 +1221,8 @@ def func_slice(parser, multi, start_index, end_index, separator=MULTI_VALUED_JOI
         end = int(end_index.eval(parser)) if end_index else None
     except ValueError:
         end = None
-    multi_var = _get_multi_values(parser, multi, separator)
-    if not isinstance(separator, str):
```

Comment:
In `_get_multi_values()`, `separator` is tested against `ScriptExpression` but here we check if it'snt an instance of `str`.
Imho it is much more correct to test if `isinstance(separator, ScriptExpression)`.
Does it deserve a patch before next release?

Reply:
I agree it is more correct, yes. But I don't think it warrants a patch. In theory this should have no other type other than `str` or `ScriptExpression`, so both statements do the same. Unless we both miss something essential. Let's play it safe and keep the code as is, don't want to introduce any last minute regression :)